nixos/modules/virtualisation/podman.nix

```diff
@@ -120,16 +120,11 @@ in
     }
     (mkIf cfg.enableNvidia {
       environment.etc."nvidia-container-runtime/config.toml".source =
-        toml.generate "config.toml" {
-          disable-require = true;
-          nvidia-container-cli = {
-            environment = [ ];
-            ldcache = "/tmp/ld.so.cache";
-            load-kmods = true;
-            ldconfig = "@${lib.getBin glibc}/bin/ldconfig";
-            no-cgroups = true;
-          };
-        };
+        pkgs.runCommandNoCC "" { } ''
+          cp ${../../../pkgs/applications/virtualization/nvidia-docker/podman-config.toml} \
+            $out/config.toml
+          substituteInPlace $out/config.toml --subst-by-var glibcbin ${lib.getBin pkgs.glibc}
```

Comment:
Move this to `nvidia-docker` with the docker `cp` and `substituteInPlace` then we should only need to source it here like the docker module does.

https://github.com/NixOS/nixpkgs/blob/e7fe577d9803885d1191c6612b95c246cb605dde/pkgs/applications/virtualization/nvidia-docker/default.nix#L62-L63

https://github.com/NixOS/nixpkgs/blob/9a63b3d3d61ca822aceeb4a994e8ef21a7593cc7/nixos/modules/virtualisation/docker.nix#L218-L220

Reply:
Done. Thanks for your patience.